packages/react-core/src/components/Page/PageBreadcrumb.tsx

```diff
@@ -28,6 +28,8 @@ export interface PageBreadcrumbProps extends React.HTMLProps<HTMLElement> {
   hasShadowBottom?: boolean;
   /** Flag indicating if the PageBreadcrumb has a scrolling overflow */
   hasOverflowScroll?: boolean;
+  /** Adds an accessible name to the breadcrumb section. Required when the has overflow scroll prop is set to true. */
```

Comment:
Nit: I know prop descriptions should use sentence case, but I'm wondering if that should apply when referencing other props?

Reply:
I'm not positive about that myself. I think that having other props in camelCase makes things a bit more clear, but it does conflict with the sentence case prop descriptions guidance as you mentioned. I realize now that some of the other prop descriptions here are already using camelCase.

@tlabaj do you have any thoughts on this?

Reply:
Same would apply for things like "onChange" and whether that should be written as "on change..." instead.